circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
```

Comment:
Not sure really... In the past (2y ago) I've chose CircleCI for their speed, Docker support and price. And since then I'm with them. 

But.. TravisCI is OK too - would you prefer this one? Can adjust.